schedule/yast/skip_registration/skip_registration_x86_64.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name:           skip_registration
+description:    >
+  Skipping registration for SLE 15, as requires network connection.
+  This is default behavior for SLE 12.
+vars:
+  SCC_REGISTER: 'none'
+  ADDONS: all-packages
+  YUI_REST_API: 1
+schedule:
+  registration:
+    - installation/registration/skip_registration
+  extension_module_selection:
+    - installation/module_selection/select_module_desktop
+  add_on_product: []
+  add_on_product_installation:
+    - installation/add_on_product_installation/accept_add_on_installation
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  system_preparation:
```

Comment:
let's validate default target as well.

Reply:
added

Reply:
same, I'm referring to the test module, not the test data which I think it is not needed.

Reply:
changed.

Reply:
@jknphy I've redressed all your comments and please take a look again. thanks